CHANGES/2396.doc

```diff
@@ -1 +1 @@
-Improve tutorial docs, using literalinclude to link to the actual files.
+Improve tutorial docs, using `literalinclude` to link to the actual files.
```

Comment:
This change is unrelated.

Reply:
It's a fix for already committed CHANGES.
`literalinclude` is a spelling error without backticks.